core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -387,12 +414,27 @@ private[spark] class TaskSchedulerImpl(
     ResourceUtils.resourcesMeetRequirements(resourcesFree, resourcesReqsPerTask)
   }
 
+  def minTaskLocality(l1: Option[TaskLocality], l2: Option[TaskLocality]) :
+  Option[TaskLocality] = {
```

Comment:
nit:
```
def minTaskLocality(
    l1: Option[TaskLocality],
    l2: Option[TaskLocality]): Option[TaskLocality] = {
 .....
}
```

Reply:
thanks for pointing out difference. don't wanna break style 👍